marmot.go

```diff
@@ -152,6 +157,10 @@ func main() {
 			}
 
 			os.Exit(0)
+		case <-reloadCh:
+			if err := replicator.ReloadCertificates(); err != nil {
```

Comment:
I wonder if we should pull this whole thing out into a method, because I can tell the moment I let this SIGHUP idea in, people would want to reload more things (refresh tables list after schema migration etc). I am still debating in my head ideas on schema migration and one of the ideas I have is to use SIGHUP to live reload schema changes. Let's extract this out into a method before we start considering anything else.